templates/template_vars.js

```diff
@@ -59,14 +59,14 @@ module.exports = function(argv, data, that){
     "qxpath" : {
       "description" : "the absolute path to the qooxdoo folder",
       "value" : argv.qxpath ? path.normalize(argv.qxpath) : undefined,
-      "default" : path.join(process.cwd(), "qooxdoo", "framework" ),
+      "default" : data.qooxdoo_path,
```

Comment:
This doesn't work because of https://github.com/qooxdoo/qooxdoo-cli/blob/master/lib/qxcli/commands/Create.js#L137 - but you could set the default path there. The current solution is fragile  and has all kind of side effects when using the CLI differently. Please always also test `qx create foo -I` (non-interactively) to  see if it still works.

Reply:
In case qxpath is set i use this for qooxdoo_path now. So there is a default in this case.
If default is set to undefined the default value is empty

Reply:
Ok! As long as it works both for interactive and non-interactive use...

Reply:
Not in line 163. There you need the value.